Presentation.Web/Controllers/API/V2/External/Organizations/OrganizationV2Controller.cs

```diff
@@ -96,7 +101,7 @@ public class OrganizationV2Controller : ExternalBaseController
         /// <returns>A list of organizations</returns>
```

Comment:
Comment is also wrong, it indicates that this endpoint returns a list

Reply:
nice catch